plugins/Plugins.props

```diff
@@ -36,7 +36,7 @@
             <SupportJustMyCode>false</SupportJustMyCode>
         </ClCompile>
         <Link>
-            <AdditionalDependencies>ProcessHacker.lib;thirdparty.lib;ntdll.lib;noenv.obj;noarg.obj;%(AdditionalDependencies)</AdditionalDependencies>
+            <AdditionalDependencies>$(SolutionDir)..\bin\$(Configuration)$(PlatformArchitecture)\ProcessHacker.lib;thirdparty.lib;ntdll.lib;noenv.obj;noarg.obj;%(AdditionalDependencies)</AdditionalDependencies>

```

Comment:
The lib file is included from the Plugin SDK path... \build\build_sdk.cmd will copy the lib file (also happens when compiling processhacker.sln in the post-build event).